action/MoveGroupSequence.action

```diff
@@ -12,3 +12,9 @@ MotionSequenceResponse response
 
 # The internal state that the move group action currently is in
 string state
+
+# The number of items in request successfully planned. Currently unused.
+int32 n_planned_items
+
+# Total number of items in request. Currently unused.
+int32 n_total_items
```

Comment:
Drop the unused statement. The fields will be used in your upcoming PR.
Further, I suggest to change names slightly:
```suggestion
# The number of processed items in request.
int32 finished_items

# Total number of items in request.
int32 total_items
```

Reply:
@rhaschke I've made most of those changes. I'm not sure I agree with dropping the n_ prefix, as I think it's nice to indicate that it's a number of things (e.g. finished_items could suggest that it contains the items themselves). Not too fussed though, since it's fairly obvious from the type.